common/djangoapps/entitlements/tests/factories.py

```diff
@@ -0,0 +1,15 @@
+import factory
+import uuid
+
+from entitlements.models import CourseEntitlement
+from student.tests.factories import UserFactory
+
+
+class CourseEntitlementFactory(factory.django.DjangoModelFactory):
+    class Meta(object):
+        model = CourseEntitlement
+
+    course_uuid = uuid.uuid4()
+    mode = 'verified'
```

Comment:
Similarly to clinton's comment, it would be good to use a random choice selector to choose various valid modes we support